template-transformer/handler.lua

```diff
@@ -133,15 +142,15 @@ function TemplateTransformerHandler:body_filter(config)
                                                                                            body = body,
                                                                                            raw_body = raw_body,
                                                                                            status = ngx.status}
-
+      ngx.log(ngx.ERR, string.format("BEFORE PREPARE :: %s", cjson_encode(transformed_body)))
       local transformed_body_json = prepare_body(transformed_body);
 
       ngx.arg[1] = transformed_body_json
 
       local json_transformed_body = cjson_decode(transformed_body_json)
       utils.hide_fields(json_transformed_body, config.hidden_fields)
 
-      ngx.log(ngx.DEBUG, string.format("Transformed Body :: %s", cjson_encode(json_transformed_body)))
+      ngx.log(ngx.ERR, string.format("Transformed Body :: %s", cjson_encode(json_transformed_body)))
```

Comment:
This is not an error, shouldn't this be an `INFO` ?

```suggestion
      ngx.log(ngx.INFO, string.format("Transformed Body :: %s", cjson_encode(json_transformed_body)))
```